contracts/core/tokens/SetToken.sol

```diff
@@ -90,22 +89,10 @@ contract SetToken is
         );
 
         // Confirm an empty _components array is not passed
-        require(
-            componentCount > 0,
-            "SetToken.constructor: Empty components array"
-        );
-
-        // Confirm an empty _quantities array is not passed
-        require(
```

Comment:
Where'd the validation for `_quantities` go?

Reply:
The `validateEqualLength` implicitly validates the unit - since the component length needs to be  > 0 and they both need to be the same